File: app/src/main/java/com/pk/dexloadexemplar/DLog/DLog.java
package com.pk.dexloadexemplar.DLog;

import android.util.Log;

public class DLog {
    public static void d(String className, String functionName, String message) {
        Log.d("DexLoadExemplar " + className + ":" + functionName, message);
    }

    public static void e(String className, String functionName, String errorMessage) {
        Log.e("DexLoadExemplar " + className + ":" + functionName, errorMessage);
    }

    public static void i(String className, String functionName, String errorMessage) {
        Log.i("DexLoadExemplar " + className + ":" + functionName, errorMessage);
    }
}

File: app/src/main/java/com/pk/dexloadexemplar/DLog/DLog.kt
package com.pk.dexloadexemplar.DLog

import android.util.Log

object DLog {
    fun d(className: String, functionName: String, message: String?) {
        Log.d("DexLoadExemplar $className:$functionName", message!!)
    }

    fun e(className: String, functionName: String, errorMessage: String?) {
        Log.e("DexLoadExemplar $className:$functionName", errorMessage!!)
    }

    fun i(className: String, functionName: String, errorMessage: String?) {
        Log.i("DexLoadExemplar $className:$functionName", errorMessage!!)
    }
}